attunement.lic

```diff
@@ -24,9 +25,13 @@ class Attunement
       return
     end
 
-    room_list = if @stationary_skills_only || DRStats.necromancer?
+    room_list = if @stationary_skills_only || (DRStats.necromancer? and !@force_power_walk)
                   [Room.current.id]
                 else
+                  if DRStats.necromancer? and @force_power_walk
+                    put "prep eotb"
+                    bput "cast", "You whisper"
```

Comment:
Negative. I viewed this as a more high level item since attunement can be learned in combat much easier and only at higher end attunements do you have to supplement with power walk.

It's a good suggestion and other commentors above have suggested using waggle_sets.  I'm planning on adding that in. 

Thank you!